src/routes/email/email.service.ts

```diff
@@ -23,7 +23,7 @@ export class EmailService {
   }): Promise<void> {
     return this.repository.resendEmailVerification({
       ...args,
-      signer: args.account,
+      account: args.account,
     });
```

Comment:
I think this can be simplified to:
```javascript
return this.repository.resendEmailVerification(args);
```

Reply:
Good catch! Addressed in [6990105](https://github.com/safe-global/safe-client-gateway/pull/944/commits/6990105ff17819e91620f80a63d99667b1dca99f)